### ISSUE TITLE
Precompile femtocleaner during auto-deployment.

### ISSUE THREAD

**Issue by Keno** [JuliaComputing/FemtoCleaner.jl#83]
Right now we time out several events at the beginning as new code paths are triggered and femtocleaner compiles itself. It would be nice to precompile once during auto-deployment, before we kill the old server.

**Comment by KristofferC** [comment]
Could also put CSTParser.jl and Deprecations.jl in sysimg with generated precompile files as well?

**Comment by Keno** [comment]
Yes, my ideal workflow here is that we pull down a new manifest, run that through something like PackageCompiler.jl and then just execute that.
